dxr/plugins/js/analyze_js/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "js_esprima",
+  "description": "esprima-based javascript plugin for DXR",
+  "author": "Mozilla",
+  "version": "1.0.0",
+  "engine": "node >= 0.8.x",
```

Comment:
I betcha it needs at least node 6.0.0 (when ES6 support was introduced), since it uses [ES6 method properties](http://es6-features.org/#MethodProperties),